var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -352,6 +363,7 @@ class Trilinos(CMakePackage):
     depends_on('superlu-dist', when='+superlu-dist')
     depends_on('superlu-dist@:4.3', when='@11.14.1:12.6.1+superlu-dist')
     depends_on('superlu-dist@4.4:5.3', when='@12.6.2:12.12.1+superlu-dist')
+    depends_on('superlu-dist@5.4:6.2.0', when='@12.12.2:13.0+superlu-dist')
```

Comment:
Are  you sure about this? I got build errors when building with `+amesos` and newer versions of superlu.

Reply:
This one is in your official release: https://github.com/spack/spack/blob/4d586c598b758210608a9cf6275cd0f1ec10d3e3/var/spack/repos/builtin/packages/trilinos/package.py#L355 I think I somehow pick that up when rebasing my local repo

Reply:
> @Char-Aznable Spack CI doesn't test the validity of recipes (although I think there are efforts underway for "smoke tests" that will verify to first order xSDK/E4S installations). I notice that your spec has almost nothing enabled except for `+tpetra+suite-sparse`. Can you try a build with the default trilinos variants (make sure your packages.yaml isn't disabling them), plus cuda? The idea being that someone who tries `spack install trilinos+cuda` should either get a spack conflict message or a working build.

This PR already takes my machines quite a few hours to build and I currently can't afford building everything of Trilinos. Most importantly, if there is no public pipeline to test the build , then I am not sure if my assertion of certain spec builds successfully for me actually mean anything to others. Someone can always jump in saying that this doesn't work for them. Not that I am not willing to do such test locally but I want to know what the end is for this.

Reply:
This is now removed